python/lsst/ts/MTAOS/wavefront_collection.py

```diff
@@ -97,15 +108,16 @@ def clear(self):
         """Clear the collection."""
 
         self._collectionData.clear()
-        self._collectionDataTaken = []
+        self._collectionDataTaken = dict()
+        self._numDataTaken = 0
 
     def getListOfWavefrontErrorAvgInTakenData(self):
         """Get the list of average wavefront error in taken data.
 
         Returns
         -------
-        list[lsst.ts.wep.ctrlIntf.SensorWavefrontData]
-            List of average wavefront error data.
+        wfe_avg: `dict`
```

Comment:
I think for the return value, the `numpy` doc string only needs the data type. Right?
```
`dict`
```